### PR TITLE
Add record group description level, refs #12367

### PR DESCRIPTION
**Pull request by mcantelon**
Add new term representing a new level of description: record group.

### CODE REVIEW

File: lib/task/migrate/migrations/arMigration0168.class.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+class arMigration0168
+{
+  const
+    VERSION = 168, // The new database version
+    MIN_MILESTONE = 2; // The minimum milestone required
+
+  public function up($configuration)
+  {
+    // Create new term for record group level of description
+    $term = new QubitTerm;
```
Comment (sbreker) [inline lib/task/migrate/migrations/arMigration0168.class.php]: Do we need to use bumpTerm here or check for existence first?

Comment (mcantelon) [reply lib/task/migrate/migrations/arMigration0168.class.php]: Good thinking... I'll add a check for existence.

Comment (mcantelon) [reply lib/task/migrate/migrations/arMigration0168.class.php]: I've now added a check for existence (I don't think bumpTerm is needed as it doesn't need a fixed ID).

**Review by sbreker — APPROVED** [review]
Looks great @mcantelon! Thanks!

**Pull request merged**